app/src/theme/index.js

```diff
@@ -377,8 +377,8 @@ const theme = {
   },
   fixedHeightModal: {
     content: {
-      height: '510px',
-      width: '382px',
+      minHeight: '510px',
```

Comment:
Lets move towards the modals adapting the height based on the inner components and not set min-heights.

Reply:
got it. FYI before i made this change it was set to `height: 510px`. should i change the name of this variable now that it's the opposite of 'fixed height'? confusing naming